csharp/src/Microsoft.Azure.EventHubs/EventHubClient.cs

```diff
@@ -15,7 +15,7 @@ public abstract class EventHubClient : ClientEntity
     {
         EventDataSender innerSender;
 
-        internal EventHubClient(ServiceBusConnectionSettings connectionSettings)
+        internal EventHubClient(EventHubsConnectionSettings connectionSettings)
```

Comment:
>EventHubs [](start = 32, length = 9)

general question, can you verify that the correct naming is EventHub"s" rather than EventHub?